cmake/developer_package/api_validator/api_validator.cmake

```diff
@@ -23,7 +23,7 @@ can't find Windows SDK version. Try to use vcvarsall.bat script")
         endif()
     endif()
 
-    set(PROGRAMFILES_ENV "ProgramFiles\(X86\)")
+    set(PROGRAMFILES_ENV "ProgramFiles\(X86\)" PARENT_SCOPE)
```

Comment:
it's not used outside of this function, do we need to move to parent scope ?

What issue do you experience?

Reply:
FYI https://github.com/openvinotoolkit/openvino/pull/25252